src/App.jsx

```diff
@@ -14,33 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ABC = 'abc';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
Also you can add default sorting in constant and use it as initial params in useState